routes/beers.js

```diff
@@ -0,0 +1,18 @@
+var express = require('express');
```

Comment:
why not const??

Reply:
poca attention to details, I am working on it